File: cpp/crescente.cpp
#include <iostream>

using namespace std;

int main(){
    int x, y;

    cout << "Digite dois numeros:" << endl;
    cin >> x >> y;

    while(x!=y){
        if(x > y){
            cout << "DECRESCENTE!" << endl;
        }
        else{
            cout << "CRESCENTE!" << endl;
        }

        cout << "Digite outros dois numeros:" << endl;
        cin >> x >> y;
    }

	return 0;
}

File: python/crescente.py
x: int; y: int

print("Digite dois numeros:")
x = int(input())
y = int(input())

while x != y:
	if x > y:
		print("DECRESCENTE!")
	else:
		print("CRESCENTE!")

	print("Digite dois numeros:")
	x = int(input())
	y = int(input())